dataeng/jobs/analytics/SnowflakeQueueDepth.groovy

```diff
@@ -0,0 +1,75 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+
+class SnowflakeQueueDepth {
+"""
+This job runs every 10 minutes using the Snowflake task automation user.
+"""
+    public static def job = { dslFactory, allVars ->
+
+        Map SnowflakeQueueDepthConfig = [
+            NAME: "collect-queue-depth",
+            MANUAL: false,
+            SCRIPT_TO_RUN: "dataeng/resources/snowflake-collect-queue-depth.sh"
+        ]
+        List jobConfigs = [
+            SnowflakeQueueDepthConfig
+        ]
+
+        jobConfigs.each { jobConfig ->
+
+            dslFactory.job(jobConfig['NAME']) {
+
+                logRotator common_log_rotator(allVars)
+                parameters secure_scm_parameters(allVars)
+                parameters {
+                    stringParam('ANALYTICS_TOOLS_URL', allVars.get('ANALYTICS_TOOLS_URL'), 'URL for the analytics tools repo.')
+                    stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), 'Branch of analytics tools repo to use.')
+                }
+                environmentVariables {
+                    env('KEY_PATH', allVars.get('KEY_PATH'))
+                    env('PASSPHRASE_PATH', allVars.get('PASSPHRASE_PATH'))
+                    env('USER', allVars.get('USER'))
+                    env('ACCOUNT', allVars.get('ACCOUNT'))
```

Comment:
I'd prefer SNOWFLAKE_USER and SNOWFLAKE_ACCOUNT, there are way too many services flying around for which we have users for me to keep track of.